swagger_server/services/evaluations.py

```diff
@@ -0,0 +1,74 @@
+from operator import attrgetter
+
+from swagger_server.models.evaluation import Evaluation
+from swagger_server.models.operand import Operand
+from swagger_server.models.operator import Operator
+from swagger_server.models.result import Result
+
+VALID_OPERANDS = {'+', '-', '/', '*', '**', '%', '//'}
+
+class EvaluationService(object):
+
+    def __init__(self, input_checked=True):
+        self.input_checked = input_checked
+
+    def evaluate_expression(self, evaluation_id):
+        evaluation = Evaluation.get_by_id(evaluation_id)
+        evaluation.status = Evaluation.EVALUATING
+        evaluation.save()
+
+        if not evaluation:
```

Comment:
I think `get_by_id` raises a `DoesNotExist` exception so we wouldn't get here yah?